### ISSUE TITLE
Failed Row Samples not showing up in Reference Checks

### ISSUE THREAD

**Issue by bleonard** [sodadata/soda-core#2008]
soda --version
soda-core, version 3.1.5

I have this

```
checks for stg_salesforce__opportunity:
  - missing_count(opportunity_id) = 0
  - values in (account_id) must exist in stg_salesforce__account (account_id)
```

There is one row that fails this. I expect to see something like this (or maybe the whole row)

```
[17:00:42] 1/12 checks FAILED:
[17:00:42]     stg_salesforce__opportunity in airbyte_warehouse_staging
[17:00:42]       values in (account_id) must exist in stg_salesforce__account (account_id) [FAILED]
[17:00:42]         values: "ID_OF_MY_MISSING_ACCOUNT"
```

But I get:

```
[17:00:42] 1/12 checks FAILED:
[17:00:42]     stg_salesforce__opportunity in airbyte_warehouse_staging
[17:00:42]       values in (account_id) must exist in stg_salesforce__account (account_id) [FAILED]
[17:00:42]         value: 1
```

This is obviously the count returned from the query.

The [documentation](https://docs.soda.io/soda-cl/reference.html#failed-row-samples) suggests that I should see the bad values. I have tried `samples limit` and `samples columns` to no avail. Please advise on the best way to see the offending data.

**Comment by tools-soda** [comment]
SAS-2801

**Comment by bastienboutonnet** [comment]
Hi @bleonard I believe that your question stems for a mis-understanding/unclear expectations set out by our docs.

The behaviour you see in the CLI is expected. The failed rows should however be pushed into soda cloud for visualisation and if you are not a user of Soda Cloud, you might want to look at this section of the docs: https://docs.soda.io/soda-cl/failed-rows-checks.html#configure-a-failed-row-sampler which explains how to get failed rows send somewhere else than Soda Cloud.

If you're a Soda Cloud user, could you let us know if the failed rows aren't showing up there? If they're not then there's defintely a bug.

**Comment by bastienboutonnet** [comment]
@bleonard I'll relay the info on making this a little bit more salient in the docs to @janet-can . As far as I know, no one added this at the last minute so I guess we really have an opportunity to make it a bit more salient.

With regards to your second point. I don't know exactly what you tried so it's hard for me to tell you. I'm going to need more info. Or maybe could you rephrase what you mean by you cannot use the `soda` command. Do you mean you cannot use soda via the CLI because you're leveraging the custom sampler pattern as explained [here](https://docs.soda.io/soda-cl/failed-rows-checks.html#configure-a-failed-row-sampler)?
